docs/getting-started/first-service.md

```diff
@@ -14,11 +14,10 @@ Since our "Hello world" Service is being deployed as a Knative Service, not a Ku
     --image gcr.io/knative-samples/helloworld-go \
     --port 8080 \
     --env TARGET=World \
-    --revision-name=world
+    --revision-name=world # (1)
```

Comment:
I found these annotations a bit hard to spot. I'm wondering whether we can make them easier to see - like in the Red Hat docs:
<img width="768" alt="Screen Shot 2022-02-07 at 12 48 53" src="https://user-images.githubusercontent.com/33425661/152797590-0538c810-9217-4971-9aa3-dcd5ed17c89b.png">

Reply:
Also, are we able to get numbering for the annotations?

Reply:
looks like numbers are possible: https://squidfunk.github.io/mkdocs-material/reference/code-blocks/?h=code+annot#annotations-with-numbers

Don't see anything yet about how to change their appearance